addons/resource.language.en_gb/resources/strings.po

```diff
@@ -10715,7 +10715,10 @@ msgctxt "#20381"
 msgid "Specials"
 msgstr ""
 
-#empty string with id 20382
+#: xbmc/video/videodatabase.cpp
```

Comment:
This is also used in VideoInfoScanner.cpp.

But maybe add a helper to CBookmark that retrieves the proper string for you so that it can be used in all places.